Working on HTML file 😎

xiacodes opened the pull request:
What I have done:

- **I organized the HTML file a bit** - there were a few mistakes such as the indentation of the code and the placement of some elements in the wrong places such as the `<head>` tag.

xiacodes commented:
Remember: Do not click `Merge pull request` until we are happy with the work 😳❗❗

The pull request is a way for me to make changes but not ready to add the work to the main code, just in case of errors and mistakes 👍🏾

louisclarencepeter reviewed (commented):
I am so happy with the updates! I learned something new already :)

xiacodes commented:
Had a busy day today - will start working towards the project more tomorrow! 👍🏾 

Will work on:
- [ ] Start getting the main content in the right place on the page
- [ ] Work on the CSS a bit

louisclarencepeter commented:
No worries! I am learning a lot from you ☺️Sent from my iPhoneOn 14. Dec 2022, at 12:36 AM, Isabelle (Loariya-Fiaba) Zennah ***@***.***> wrote:﻿
Had a busy day today - will start working towards the project more tomorrow! 👍🏾
Will work on:

 Start getting the main content in the right place on the page
 Work on the CSS a bit


—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

xiacodes commented:
Commit #74002b4 info:

- I changed the `<address>` tag to an `<ul>` list because it would be better since you are _listing_ information

xiacodes commented:
Commit #[8b2c33d](https://github.com/louisclarencepeter/Portfolio/pull/1/commits/8b2c33d31cd207a6796e833879c246af31b7e56f) info:

- Only download fonts to you project that _isn't_ available online - that you can't import from a website
- Example of it not being available, through `@import` in CSS, is if you paid and downloaded a font you want to use on your website
- '**_Space Mono_**' is a Google font, which can be accessed from the [Google Font](https://fonts.google.com/)
- All you had to do is select the font and copy and paste the code generated to the `.css` file - this is what I did in this commit

xiacodes commented:
Commit #[394173b](https://github.com/louisclarencepeter/Portfolio/pull/1/commits/394173bfce79e9f8dc38a87649aec2b398545baa) info:

- Added some styling to the `<body>` in the CSS file
- Added some styling to the `.container` in the CSS file
- Renamed the `<div>` classes for the skills section according to the title of the coding language

xiacodes commented:
Commit #[cc472bb](https://github.com/louisclarencepeter/Portfolio/pull/1/commits/cc472bb790a0cdb2d4e4ba37cc1b291c456977bf) info:

- I fixed the head title and image
- **PRO-TIP:** To make two element sit next to each other but apply space between them, do the following:
> 1. Put the two elements in one `<div>` in HTML
> 2. Apply the following CSS styling to that div:  `display: flex; justify-content: space-between; align-items: center;`

xiacodes commented:
Commit #[2796ca0](https://github.com/louisclarencepeter/Portfolio/pull/1/commits/2796ca0c8e93d91a4d79f26cd762204fe9ed45fa) info:

- When you have links on a page, it's best to anchor them to the link's location using the `<a href='#'> </a>` tag
- Makes it easier for the person to click to the link, then having to copy and paste it in the search bar
- **The CSS behind the link:** I gave the links a colour when you hover, just to indicate that t is a clickable link to the user

xiacodes commented:
Commit #[f29cce4](https://github.com/louisclarencepeter/Portfolio/pull/1/commits/f29cce4e309b2d745c2113e324cb13bcde680ae1) info:
- You had the correct idea using `display: grid`, `grid-template-columns` and `gap` to had the table for the skills
- Try to not over complicate things with the grids
- You have 3 simple columns, situations like this just use the values: `1fr 1fr 1fr` in `grid-template-columns`
- `1fr 1fr 1fr` Just means "split the grid into 3 columns evenly" 
- It will set the width for you automatically, which you tried to do the way you did it would lead to issues when viewing the page on different screens
- Learn more about the `fr` CSS unit here - [**LINK**](https://css-tricks.com/introduction-fr-css-unit/)

xiacodes commented:
Commit #[c6203d7](https://github.com/louisclarencepeter/Portfolio/pull/1/commits/c6203d7208257e6cb43372a4e6acd4697b25b36a) info:
- I deleted the screenshots of the progression bars - you can make one using HTML + CSS3
- Added a `<div>` and gave it a class named `myProgress` - this acts as the white parts of the bar
- Inside that `<div class="myProgress">`, I made another  `<div>` and gave it a class named `myBar` - this will act as the base of the coloured parts of the bar
- Because you are at different stages of each language, you need to fill the bar at different points
- I gave each `<div class="myBar">` and extra class inside; `htmlBar`, `cssBar` and `javascriptBar` 
- Each of those classes have different `width` percentage values
- This helps in the future when you want to come back and update the progress bar
- E.g. JavaScript changes to being 6/10, go to the CSS file and change the `width: 0%;` to `width: 60%;` and the page will update!

xiacodes commented:
Commit #[4f024a0](https://github.com/louisclarencepeter/Portfolio/pull/1/commits/4f024a0a4220c0e08c4487a0d206a4d8d14bda17) info:
- To get shapes on a page, read more here: [Working with Shapes in Web Design](https://css-tricks.com/working-with-shapes-in-web-design/)
- To get the shapes to move to certain places on the screen but not affect the other elements, read more here: [CSS Layout - The position Property](https://www.w3schools.com/css/css_positioning.asp)
- Honestly, I could explain how I did it, but I don't even know how I did it but it seems to work 😂

xiacodes commented:
## Update
Done for today. I coded a big chunk of the webpage now! What I did today:

- [x] Fixed the Contact Info section
- [x] Fixed the header title and image section
- [x] Fixed the Language Category table
- [x] Worked on the School and Progress section
- [x] Added HTML+CSS progress bar instead of an image screenshot
- [x] Added the white box graphics to the page, just like in the wireframe

Tomorrow I will work on the responsiveness of the page using `@media` queries. What I plan to do tomorrow

1. Make the page responsive on big screens
2. Make the page responsive on medium screens
3. Make the page responsive on small screens
